packages/braid-design-system/src/lib/components/private/InlineField/StyledInput.tsx

```diff
@@ -175,18 +195,18 @@ export const StyledInput = forwardRef<
           id={id}
           name={name}
           value={value}
-          onChange={
-            isMixed
-              ? (e: ChangeEvent<HTMLInputElement>) => {
-                  if (ref && typeof ref === 'object' && ref.current) {
-                    ref.current.indeterminate = indeterminateRef.current;
-                  }
-                  if (typeof onChange === 'function') {
-                    onChange(e);
-                  }
-                }
-              : onChange
-          }
+          onChange={(e: ChangeEvent<HTMLInputElement>) => {
```

Comment:
Is this type annotation still necessary? Would've though it would already be inferred as this type.
```suggestion
          onChange={(e: ChangeEvent<HTMLInputElement>) => {
```

Reply:
Unfortunately it gets inferred as a `HTMLElement` and we require it to be a `HTMLInputElement` explicitly as we reference the `indeterminate` property.